app/components/channel-header/component.js

```diff
@@ -1,8 +1,24 @@
 import Ember from 'ember';
+import addMonths from 'npm:date-fns/add_months';
+import isAfter from 'npm:date-fns/is_after';
 
-const {Component, computed} = Ember;
+const {
+	Component,
+	computed,
+	get
+} = Ember;
 
 export default Component.extend({
 	classNames: ['ChannelHeader'],
-	hasLinks: computed.or('channel.link', 'channel.hasCoordinates')
+	hasLinks: computed.or('channel.link', 'channel.hasCoordinates'),
+	shouldShowUpdated: computed('channel.updated', function() {
+		const updatedTimestamp = get(this, 'channel.updated');
+
+		if (!updatedTimestamp) {
+			return
```

Comment:
use `return false` so it the return value is consistent

Reply:
sure